.github/workflows/tx-hammer-plugins.yml

```
@@ -0,0 +1,93 @@
+name: Translations
+
+on:
+  workflow_call:
+    inputs:
+      plugin_name:
+        description: 'Name of the Hammer plugin. (Not the repository name)'
+        required: true
+        type: 'string'
+    secrets:
+      tx_token:
+        description: 'API token for the Transifex.'
+        required: true
+      gh_token:
+        description: 'GitHub token with permissions to create a pull request'
+        required: true
+env:
+  branch_name: "tx-${{ github.run_id }}"
+jobs:
+  tx-update:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ${{ inputs.plugin_name  }}
+
+    steps:
+      - name: Get hammer-cli
+        uses: actions/checkout@v4
+        with:
+          repository: theforeman/hammer-cli
+          ref: master
+          path: hammer-cli
+
+      - name: Get ${{ github.repository }}
+        uses: actions/checkout@v4
+        with:
+          path: ${{ inputs.plugin_name  }}
+      - name: Install Transifex dependencies
+        run: |
+          sudo apt-get -y install gettext
+
+      - name: "Install Transifex"
+        run: |
+          curl -o- https://raw.githubusercontent.com/transifex/cli/master/install.sh | bash
+          sudo mv tx /usr/local/bin/
+          sudo chown -R runner:runner /usr/local/bin
+
+      - name: Configure plugin
+        run: |
+          echo "gemspec path: '../hammer-cli', name: 'hammer_cli'" > Gemfile.local.rb
+          mkdir -p ./.tx
+          echo "[main]" > ./.tx/config
+          echo "host = https://rest.api.transifex.com" >> ./.tx/config
+          echo "token = ${{ secrets.tx_token  }}" >> ./.tx/config
```

I'd consider the multi line statement
```suggestion
          cat > ./.tx/config <<EOF
          [main]
          host = https://rest.api.transifex.com
          token = ${{ secrets.tx_token  }}
          EOF
```